core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.component.spec.ts

```diff
@@ -245,9 +244,7 @@ describe('Skills List Component', () => {
   beforeEach(waitForAsync(() => {
     TestBed.configureTestingModule({
       imports: [
-        MatCardModule,
-        MatCheckboxModule,
-        MatRadioModule,
+        MaterialModule,
```

Comment:
Can we import the specific modules required rather than import a larger module that may contain components we do not need?

Reply:
Yes, I have updated it, but can you please help me to understand its advantages over using a single `MaterialModule`?
Thank you